internal/pkg/cli/run_local.go

```diff
@@ -301,47 +320,28 @@ func (o *runLocalOpts) Execute() error {
 		fmt.Printf("hosts: %+v\n", hosts)
 	}
 
-	mft, _, err := workloadManifest(&workloadManifestInput{
-		name:         o.wkldName,
-		appName:      o.appName,
-		envName:      o.envName,
-		ws:           o.ws,
-		interpolator: o.newInterpolator(o.appName, o.envName),
-		unmarshal:    o.unmarshal,
-		sess:         o.envManagerSess,
-	})
-	if err != nil {
-		return err
-	}
-
-	containerURIs, err := o.buildContainerImages(mft)
-	if err != nil {
-		return fmt.Errorf("build images: %w", err)
-	}
-
-	// replace built images with the local built URI
-	for name, uri := range containerURIs {
-		ctr, ok := task.Containers[name]
-		if !ok {
-			return fmt.Errorf("built an image for %q, which doesn't exist in the task", name)
-		}
-
-		ctr.ImageURI = uri
-		task.Containers[name] = ctr
-	}
-
 	sigCh := make(chan os.Signal, 1)
 	signal.Notify(sigCh, syscall.SIGINT, syscall.SIGTERM)
 
 	errCh := o.orchestrator.Start()
 	o.orchestrator.RunTask(task)
 
+	stopCh := make(chan struct{})
+	watchCh := make(chan interface{})
+	watchErrCh := make(chan error)
```

Comment:
i'm not sure why this function creates the channels that `watchLocalFiles` ends up using 🤔 typically, the goroutine that _writes_ to the channels should create and close them